maskit/masked_parameters.py

```diff
@@ -72,15 +85,24 @@ def perturb(self, amount: int = None, mode: PerturbationMode = PerturbationMode.
         elif self.perturbation_axis == PerturbationAxis.RANDOM:
             self._perturb_random(amount, mode, random)
         else:
-            raise ValueError(f"The perturbation {self.perturbation_axis} is not supported")
-
-    def _perturb_layers(self, amount: int = None, mode: PerturbationMode = PerturbationMode.INVERT, random: bool = True):
+            raise ValueError(
```

Comment:
In case you want to support it at a later time. :)
```suggestion
            raise NotImplementedError(
```